test/Twilio.Test/Security/RequestValidatorTest.cs

```diff
@@ -31,6 +31,14 @@ public void TestValidateDictionary()
             Assert.IsTrue(_validator.Validate(Url, _parameters, signature), "Request does not match provided signature");
         }
 
+        [Test]
+        public void TestValidateDictionaryMixedCase()
+        {
+            const string signature = "RSOYDt4T1cUTdK1PDd93/VVr8B8=";
+            const string url = "https://MyCompany.com:1234/MyApp.PHP?foo=1&bar=2";
+            Assert.IsFalse(_validator.Validate(url, _parameters, signature), "Request should have failed validation but didn't");
```

Comment:
Don't we want to test the case where mixed case succeeds?

Reply:
Added more tests.

Reply:
Generating the signatures using Python via:

```python
from twilio.request_validator import RequestValidator

validator = RequestValidator("12345")
uri = "http://username:password@MyCompany.com/myapp.php?foo=1&bar=2"
params = {
    "CallSid": "CA1234567890ABCDE",
    "Digits": "1234",
    "From": "+14158675309",
    "To": "+18005551212",
    "Caller": "+14158675309",
}
print(validator.compute_signature(uri, params, utf=False))
```